configure.ac

```diff
@@ -1046,6 +1046,16 @@ PHP_ARG_ENABLE([undefined-sanitizer],,
     [Enable undefined sanitizer])],
   [no],
   [no])
+PHP_ARG_ENABLE([async-api],,
+  [AS_HELP_STRING([--disable-async-api],
+    [Disable async API support])],
+  [yes],
+  [no])
+
+AS_VAR_IF([PHP_ASYNC_API], [yes], [
+  AC_DEFINE([PHP_ASYNC_API], [1], [Define to 1 if async API support is enabled.])
+])
+
```

Comment:
I think it should be always enabled so I would remove this and all the `PHP_ASYNC_API` ifdefs...